docker/quarto/docker-compose.build.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  quarto:
+    pull_policy: build
+    image: ${VAST_CONTAINER_REGISTRY:-docker.io}/tenzir/vast-quarto:${VAST_CONTAINER_REF:-latest}
```

Comment:
We probably wanna build nd upload this one in CI, which afaict is still missing from this PR. Is that still in scope?

Reply:
Haven't added CI yet, but it is definitively on the TODO list. 

I have now a more exotic Dockerfile structure that I would like to discuss with you. Among other, I am not 100% sure that it makes much sense to upload the image.